doc/explanation/projects.md

```diff
@@ -46,6 +46,15 @@ To edit them, you must remove all instances first.
 New features that are added in an upgrade are disabled for existing projects.
 ```
 
+```{important}
+In a multi-tenant environment, all projects should have all features enabled.
```

Comment:
Can we clarify this to say that its only recommend when using non-fine grained identities?

Reply:
I think it's fine to disable features when using fine-grained identities. The administrator just needs to be aware that they will need to grant some level of permission to the group that allows them to interact with resources in the default project.

E.g.
```
lxc auth group create foo-group
lxc project create foo-project # Has networks disabled.
lxc auth group permission add foo-group project foo-project operator
lxc auth group permission add foo-group project default can_view_networks
```

`foo-group` can now create instances in `foo-project` using networks from the default project, but can't edit or delete the networks in the default project.

Reply:
exactly :) Which is why I don't think we should blanket say "In a multi-tenant environment, all projects should have all features enabled."

Reply:
Ahhh I see what you mean now :) will update

Reply:
Done.